WordPress/Classes/ViewRelated/System/Notices/NoticeView.swift

```diff
@@ -209,3 +270,40 @@ class NoticeView: UIView {
         static let shadowOffset = CGSize(width: 0.0, height: 2.0)
     }
 }
+
+fileprivate extension UIView {
+    func addTopBorder() {
```

Comment:
Its seems a little heavy handed to add these methods to all views.  Would it make sense to create a subclass just for actionBackgroundView and cancelBackgroundView rather than an extension?

Reply:
It's only to views in this file. Is that better… I don't know 😆